### DISCUSSION TITLE
A new bug tracker: bugs.xmoj-bbs.tech

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#388] [Announcements]
https://bugs.xmoj-bbs.tech

@PythonSmall-Q 你的帐号是: 	**************************

**Comment by boomzero** [comment]
@langningchen 

	chenlangning2009@outlook.com

密码: hello_world

**Comment by boomzero** [comment]
note: 与github自动同步

**Comment by PythonSmall-Q** [comment]
@boomzero 
![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/106425289/75f03cd7-257c-4fd3-9dd3-0688f6212c2f)
注意哦只有10个人的用户限制，所以考虑一下删掉不是开发组的人

**Comment by PythonSmall-Q** [comment]
![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/106425289/f50f9fd9-14eb-4394-a191-2d6d320a7ac4)
 @boomzero

**Comment by PythonSmall-Q** [comment]
超可爱的猫猫

**Reply by PythonSmall-Q** [reply]
传下去，我有少女心（？？？

**Comment by boomzero** [comment]
???

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 这个是和Sentry差不多吗？

**Comment by langningchen** [comment]
好像不是

**Comment by langningchen** [comment]
@boomzero 不用给我分配账号，我就看看，应该用不到
